packages/react-native-sdk/docusaurus/docs/reactnative/02-tutorials/02-audio-room.mdx

```diff
@@ -1,9 +1,1111 @@
 ---
 title: Audio Room Tutorial
-description: How to build an audio room using Stream's video SDKs
+description: How to build an audio room using Stream's React Native Video SDK
 ---
 
-:::info
-This guide is still in progress and will be updated soon.
-If you have any questions, please reach out to us on our [website](https://getstream.io/video/#contact).
+import { TokenSnippet } from '../../../shared/_tokenSnippet.jsx';
+
+This tutorial will teach you how to build an audio room experience like Twitter Spaces or Clubhouse.
+The end result will look like the image below and will support the following features:
+
+- Backstage mode. You can start the call with your co-hosts and chat a bit before going live
+- Calls run on Stream's global edge network for optimal latency and scalability
+- There is no cap to how many listeners you can have in a room
+- Listeners can raise their hand, and be invited to speak by the host
+- Audio tracks are sent multiple times for optimal reliability
+
+![Preview of the final result](../assets/02-tutorials/02-audio-room/audio-room-target.png)
+
+Time to get started building an audio room for your app.
+
+### Step 1 - Setup a new React Native app
+
+Create a new React Native app using the official template,
+
+```bash title=Terminal
+npx react-native@latest init StreamAudioRoomExample
```

Comment:
Are we not focusing on expo users? Asking since this is confusing to me that we have this command which would create a RN CLI app.

Reply:
Currently, we dont  really support expo of course

Reply:
Also react uses vite.. 

you need to focus on one template to create a new app

Reply:
Okay, if you think this is fine, we can have it. Rather I would also link the React Native official guide to create a new app that includes both expo and CLI.

Reply:
yeah for later though.. we cant do everything now, focus is to close this one